ui/app/services/path-help.js

```diff
@@ -224,7 +220,7 @@ export default Service.extend({
       }
       // put url params (e.g. {name}, {role})
       // at the front of the props list
-      const newProps = { ...paramProp, ...props };
+      const newProps = assign({}, paramProp, props);
```

Comment:
looks like [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#browser_compatibility) is available in all browsers, should be OK to use

Reply:
I can apply this change in later PR's. I wan to keep this PR as a purely revert with no additional logic changes.